src/FSharpPlus/Operators.fs

```diff
@@ -9,102 +9,176 @@ module Operators =
 
     // Common combinators
 
-    /// Creates a new function with first two arguments flipped.
+    /// <summary>Creates a new function with first two arguments flipped.</summary>
+    /// <category>common</category>
     let inline flip f (x: 'T) (y: 'V) : 'Result = f y x
```

Comment:
These categories are only partially done, but give a nice table of contents. It's no big issue since functions without a category get put into "Other module members":
![image](https://user-images.githubusercontent.com/41101/95645794-1ed84300-0b0a-11eb-9bae-94533fc70e48.png)